source/Calamari.Common/Plumbing/Deployment/PackageRetention/IVersionConverter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Octopus.Versioning;
+
+namespace Calamari.Common.Plumbing.Deployment.PackageRetention
+{
+    public class IVersionConverter : JsonConverter
```

Comment:
Is the name `IVersionConverter` intentional?

Reply:
Yes and no. Its a converter for an IVersion, but calling it IVersionConverter implies the whole thing is an interface.  I've fixed it now.